sdk/translation/Azure.AI.Translation.Document/src/GetDocumentStatusesOptions.cs

```diff
@@ -11,12 +11,12 @@ namespace Azure.AI.Translation.Document
     /// A class which defines filtering and ordering options
     /// for listing all document statuses for a certain translation operation.
     /// </summary>
-    public partial class DocumentFilter
+    public partial class GetDocumentStatusesOptions
```

Comment:
with this move, I wonder if we could undo something I did 😆 .
The generated client had a class called `DocumentFilter` that was used for other purposes I didn't like, so I made it internal. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/translation/Azure.AI.Translation.Document/src/DocumentFilterInternal.cs
Now that we are not using the public `DocumentFilter` class, we can undo that and let the generated `DocumentFilter` be internal and that's it.

Let me know if that is not clear and we can sync

Reply:
Sure thing. So this involves allowing it to generate DocumentFilterInternal with the name DocumentFilter the same way it used to before we defined the customized model. I just need to make sure that the generated class is internal.

Got it!

Reply:
thank you!!